ehrapy/api/preprocessing/encoding/_encode.py

```diff
@@ -18,13 +19,16 @@ class Encoder:
     """The main encoder for the initial read MuData or AnnData object providing various encoding solutions for
     non numerical or categorical data"""
 
-    available_encodings = {"one_hot_encoding", "label_encoding", "count_encoding"}
+    available_encodings = {"one_hot_encoding", "label_encoding", "count_encoding", "hash_encoding"}
```

Comment:
```suggestion
    multi_encoding_modes = {"hash_encoding"}
    available_encodings = {"one_hot_encoding", "label_encoding", "count_encoding", *multi_encoding_modes}
```

Note that the order of sets might be undefined. Maybe use a List instead of a set here. Delete the `multi_encoding_modes` below.

Reply:
Nice idea! Orders does not really matter here though so Sets are fine.